### DISCUSSION TITLE
Chat problem

### DISCUSSION THREAD

**Discussion by turykmark** [LiveHelperChat/livehelperchat#1907] [General]
Hello Live Helper Chat Team, 
I'm sorry to bother you, but after installing this incredible software, I'm having a little problem: 
-When I first open the chat on a web page, when I send the message from the interface to the website, nothing happens. It is necessary to change page or to refresh, after that all works perfectly. 
Would you have an idea for this? 
Thanks in advance

**Comment by PeopleInside** [comment]
Hi, how we can help without having more informations?
First of all we are not able to see the issue because there is no public page where look at, no screenshots, no info about PHP version used, where and how you downloaded and installed the chat.
